Check for cpp at configure time. This causes @CPP@ to be replaced with the
cpp that was found, fixing the error:

/bin/sh: CPP@: command not found
--- configure.ac.orig	2011-11-17 06:51:31.000000000 -0600
+++ configure.ac	2024-01-26 08:01:15.000000000 -0600
@@ -11,6 +11,7 @@
 
 # Checks for programs.
 AC_PROG_CC
+AC_PROG_CPP
 AC_PROG_LN_S
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
